harmony-fe/package.json

```diff
@@ -24,7 +24,8 @@
   "dependencies": {
     "@emotion/react": "^11.11.1",
     "@emotion/styled": "^11.11.0",
-    "@mui/material": "^5.14.11",
+    "@mui/icons-material": "^5.14.14",
+    "@mui/material": "^5.14.14",
```

Comment:
Changing library versions constantly might not be a good idea. Just for the record

Reply:
Needed to use icons-material. Copied the install code from Mui site but did not realise it added other libraries with it in the install. Will keep an eye out. Thanks!